modules/dashboard/css/dashboard.css

```diff
@@ -58,3 +58,18 @@
     margin-top: 10px;
     float: left;
 }
+
+/*Custom legend scrollable*/
+.legend-container span {
+  display: inline-block;
+  margin-left: 7px;
+  margin-right: 7px;
+  padding: 5px;
+}
+.legend-container div {
+  width: 100%;
+}
+.legend-container {
+  height: 300px;
```

Comment:
where does "300px" come from?

Reply:
Hi @driusan, I set the legend height to 300px because the graph is about that height. 100% doesn't work and just makes the legend keep increasing; the panel keeps increasing in height to fit the legend. I changed height to max-height because it likely makes more sense and even if the graph is greater than 300x, having the legend at 300px as max-height looks alright at the top right corner of the graph.